internal/cli/test.go

```diff
@@ -130,16 +156,16 @@ auth0 test login -c <id> --connection <connection>`,
 					ansi.Faint("Hint:"), inputs.ClientID)
 
 				if err := cli.setFirstCommandRun(inputs.ClientID, commandKey); err != nil {
+					cleanupTempApplication(isTempClient, cli, inputs.ClientID)
 					return err
 				}
 			}
-
+			cleanupTempApplication(isTempClient, cli, inputs.ClientID)
```

Comment:
Shouldn't we be calling the cleanup function once with `defer`?

Reply:
Yeah, probably similar to the cleanup func of the callbacks PR in #218 we want it to always run.

Reply:
Good call, thanks!